src/lib/wallet/GoodWalletClass.js

```diff
@@ -608,11 +607,13 @@ export class GoodWallet {
   async getReceiptWithLogs(transactionHash: string) {
     const chainId = this.networkId
     const transactionReceipt = await this.wallet.eth.getTransactionReceipt(transactionHash)
+
     if (!transactionReceipt) {
       return null
     }
 
-    const logs = abiDecoder.decodeLogs(transactionReceipt.logs).filter(_ => _)
+    const logs = abiDecoder.decodeLogs(transactionReceipt.logs).filter(identity)
```

Comment:
revert

Reply:
identity is the same as something => something ...
also we could just import filter from lodash which uses identity as predicate by default and do filter(abiDecoder.decodeLogs(......